clients/cloud/java/src/main/java/io/confluent/examples/clients/ConsumerExample.java

```diff
@@ -42,23 +49,30 @@ public static void main(String[] args) throws Exception {
     Properties props = loadConfig(args[0]);
 
     // Add additional properties.
-    props.put("enable.auto.commit", "true");
-    props.put("auto.commit.interval.ms", "1000");
-    props.put("key.deserializer", "org.apache.kafka.common.serialization.StringDeserializer");
-    props.put("value.deserializer", "org.apache.kafka.common.serialization.StringDeserializer");
-
-    // WARNING: These settings are for CLI tools / examples. Don't use in production unless you want to reset the consumer app every reboot.
-    props.put("group.id", "example" + new Random().nextInt());
-    props.put("auto.offset.reset", "earliest");
+    props.put(ConsumerConfig.ENABLE_AUTO_COMMIT_CONFIG, "true");
+    props.put(ConsumerConfig.AUTO_COMMIT_INTERVAL_MS_CONFIG, "1000");
```

Comment:
since this application runs for such a short time even this value probably won't be used, but commits will be performed when closing the consumer, so I don't think this config is really needed.

Reply:
reminder to remove this line.

Reply:
K